### ISSUE TITLE
Will there be an shopware 6.5 version of this theme?

### ISSUE THREAD

**Issue by petritz** [digitalmanufaktur/shopware6-manu-theme#23]

**Comment by FlorianVater** [comment]
Would be a bummer if there won't be any update. We would love to update our shop but are stuck because of the Manu Theme

**Comment by tobiasdias** [comment]
Hey @FlorianVater , 
unfortunately there will be no further general update for the Manu Theme.
We did decide to create a new follow up theme though, which will be available approximately around September/October.
We are sorry for any inconveniences.
If you cannot change your theme and are dependend on the Manu Theme, we as the digital.manufaktur would be happy to help in any individual cases. For further details please contact us via our website https://www.digitalmanufaktur.com
Best regards!

**Comment by amenk** [comment]
@tobiasdias was there a follow up theme release in the meanwhile?

**Comment by tobiasdias** [comment]
> @tobiasdias was there a follow up theme release in the meanwhile?

@amenk there will be no follow up theme from our site, at least not an open source one that will be public.

**Comment by amenk** [comment]
Thanks for the quick response - @tobiasdias closed source / paid would be fine. Just looking for a migration path. We probably should send you an email?

While unlikely, we also consider maintaining this here, but there is no license defined :( 

https://github.com/digitalmanufaktur/shopware6-manu-theme/issues/21

**Comment by tobiasdias** [comment]
@amenk yes, please send us an email over our website mentioned above and then we can further discuss it happily!

**Comment by crysix** [comment]
Hi @amenk and @FlorianVater , I currently made an updated version of the theme for a customer... this is working with the latest 6.6 version of shopware. And also versions below should work. Hopefully there is just a last bug to fix, all other problems und bugs are already fixed and the theme is already generally working. There are only some scss compiler errors left.

**Comment by amenk** [comment]
Sounds promising, @crysix  is that in a fork? 

@tobiasdias my colleague reached out via email - but can the license of the current code be clarified on GitHub in the meantime?

**Comment by crysix** [comment]
Its no fork (yet) because it currently is only for one of our customers, who is using this theme in shopware 6.4, which we will update.

**Comment by amenk** [comment]
Hey @crysix, we talked to DMF and they said we could fork an provide an update.
Would you be willing to publish your current updated code as a fork - we are happy to help with testing and fixing remaining issues, if any.

**Comment by amenk** [comment]
Hi @crysix how is the status on your side? Are you able to publish the (eventually unfinished?) update of the theme as a fork?

**Comment by crysix** [comment]
Hi, not yet. Customer is thinking about not using shopware anymore. So if the project will be stopped here, we wont finish the fixes for current shopware version.  If he wants to continue, we will create a fork... but honestly it doesn't look good.

**Comment by amenk** [comment]
Uh, @crysix thanks for the quick response - any chance to publish the work in progress? either on GitHub or via zip & email?
(a.menk at imi.de would be my mail)
